test/integration/controllers/compaction/compaction_suite_test.go

```diff
@@ -53,10 +54,11 @@ var _ = BeforeSuite(func() {
 	intTestEnv = setup.NewIntegrationTestEnv(testNamespacePrefix, "compaction-int-tests", crdPaths)
 	intTestEnv.RegisterReconcilers(func(mgr manager.Manager) {
 		reconciler := compaction.NewReconcilerWithImageVector(mgr, &compaction.Config{
-			EnableBackupCompaction: true,
-			Workers:                5,
-			EventsThreshold:        100,
-			ActiveDeadlineDuration: 2 * time.Minute,
+			EnableBackupCompaction:    true,
+			Workers:                   5,
+			EventsThreshold:           100,
+			ActiveDeadlineDuration:    2 * time.Minute,
+			MetricsScrapeWaitDuration: 60 * time.Second,
```

Comment:
Please reduce the duration to a smaller value, so that tests run faster. I'm not sure whether the test waits for a compaction job to finish running though. Please check this once and adjust this value accordingly.

Reply:
@shreyas-s-rao The test does not run actual compaction job. So the `MetricsScrapeWaitDuration` is not actually passed to the compaction job of etcd backup restore. The test only checks if the arguments to create compaction job are correct or not. So, the time duration won't affect the test time